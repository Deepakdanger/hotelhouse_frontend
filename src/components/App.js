import React from 'react';
import Home from '../containers/Home';

const App = () => (
  <div className="app">    
    <Home />
  </div>
);

export default App;
