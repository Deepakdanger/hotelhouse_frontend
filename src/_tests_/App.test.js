/* eslint-disable react/jsx-props-no-spreading */
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { QueryClient, QueryClientProvider } from 'react-query';
import { transitions, positions, Provider as AlertProvider } from 'react-alert';
import AlertTemplate from 'react-alert-template-basic';
import App from '../components/App';
import store from '../reducers/index';

const queryClient = new QueryClient();

const options = {
  // you can also just use 'bottom center'
  position: positions.TOP_CENTER,
  timeout: 5000,
  offset: '30px',
  // you can also just use 'scale'
  transition: transitions.SCALE,
};

describe('Testing App with redux', () => {
  it('App.js snapshot testing', () => {
    const app = renderer.create(
      <Provider store={store}>
        <QueryClientProvider client={queryClient}>
          <AlertProvider template={AlertTemplate} {...options}>
            <App />
          </AlertProvider>
        </QueryClientProvider>
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
