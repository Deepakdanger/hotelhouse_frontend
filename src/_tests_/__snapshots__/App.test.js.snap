// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App with redux App.js snapshot testing 1`] = `
<div
  className="app"
>
  <div>
    Loading...
  </div>
</div>
`;
